api/1.0/controller/events/create.controller.js

```diff
@@ -28,7 +28,7 @@ export default async function eventCreateHandler(req, res, next) {
 			case ECONNREFUSED.errno:
 				return res.status(500).json({ error: ECONNREFUSED.message });
 			default:
-				break;
+				return res.status(500).json({ error: err });
```

Comment:
I'll handle this part on my branch, you can restore this change to avoid conflict